src/controllers/AdminerController.php

```diff
@@ -1,13 +1,13 @@
 <?php
 namespace Simple\Adminer\Controllers;
 
-use App\Http\Controllers\Controller;
+use Illuminate\Routing\Controller;
 
 class AdminerController extends Controller
 {
     public function __construct()
     {
-        $this->middleware('adminer');
+    	// AdminerServiceProvider::register holds the middleware register so it does not need addeed manually.
```

Comment:
No way to check if the group config exists or to catch the Exception from there.
Customization could be done via custom route as stated in the README.md update.